src/app/frontend/secret/list/card.html

```diff
@@ -32,6 +32,12 @@
         </kd-middle-ellipsis>
       </div>
     </kd-resource-card-column>
+    <kd-resource-card-column ng-if="$ctrl.secret.type">
```

Comment:
This can hide the whole column and destroy table view. Column header has no "if" and if this will be hidden, the table will be corrupted.